product_cost_security/views/product_views.xml

```diff
@@ -10,6 +10,11 @@
                     name="groups"
                 >product_cost_security.group_product_cost</attribute>
             </xpath>
+            <xpath expr="//page[@name='purchase']" position="attributes">
+                <attribute
+                    name="groups"
+                >product_cost_security.group_product_cost</attribute>
+            </xpath>
```

Comment:
```suggestion
            <xpath expr="//page[@name='purchase']" position="attributes">
                <attribute
                    name="groups"
                >purchase.group_purchase_manager</attribute>
            </xpath>
```

Although its correct, maybe it's a good idea create new addon to set groups to purchase page. What do you think?

Reply:
On the Purchase page we have the cost associated to each vendor, so I think that security group is correct... But I prefer the opinion of 1 more... What do you think @sergio-teruel ??
![image](https://user-images.githubusercontent.com/35952655/117114853-a0d0d780-ad8c-11eb-89d1-0ee85381f9b3.png)

Reply:
Or maybe just set the group to the field in the tree element

Reply:
Normally, purchase users must have permission to view the cost of the product, so I think that this is not problem.
Maybe You can try add to group_purchase_user the security cost group as implied_ids, by this way, a purchase user always will have the security_cost group.